histoqc/config/config_light.ini

```diff
@@ -18,6 +18,7 @@ steps= BasicModule.getBasicStats
 ;    BrightContrastModule.getBrightnessByChannelinColorSpace:YUV
 ;    DeconvolutionModule.separateStains
     SaveModule.saveFinalMask
+    SaveModule.saveMacro
```

Comment:
great! 

2 more questions:
1) should we add it into the other config files (in particular think 2.1 is the new default which people should use, and use that as a template)

2) is this going to be visible/selectable from the UI? i think it just involves adding the "_macro.png" suffix somewhere in JS?

Reply:
I will add **SaveModule.saveFinalMask** into config.ini and add **_macro.png** suffix in JS.